apps/taxes/src/modules/app/webhook-response.ts

```diff
@@ -14,6 +15,8 @@ export class WebhookResponse {
   }
 
   error(error: unknown) {
+    Sentry.captureException(error);
+
```

Comment:
Do we need this? I think Sentry by default should catch all thrown exceptions

Reply:
I guess sentry catches unhandled.